### ISSUE TITLE
Suggestion: Integration of other browser-based reading formats

### ISSUE THREAD

**Issue by konarster** [janeczku/calibre-web#63]
For example, CBZ/CBR for graphic novels, potentially using https://github.com/janeczku/calibre-web?

As is, this is not just so much better than the Calibre web-based interface, it's a great solution in it's own right.  Thank you!

**Comment by OzzieIsaacs** [comment]
Kthoom seems to be a good lib (support for cbr,cbz,cbt files): https://github.com/nitmir/kthoom

**Comment by pyrostic** [comment]
I'm not looking for drag n drop, I'm more looming for an interface that can pull from my library and display it in the frame of the device I'm looking st it on.

**Comment by OzzieIsaacs** [comment]
Yeah, forget the drag'n drop. You want to read comics in a browser window like it's possible for ePub files now. This is what I read in this issue

**Comment by pyrostic** [comment]
Yessir!  Shouldnt be to hard to add on support.

**Comment by gonzalo** [comment]
Same opinion here.  I would be great to be able to read my comic books directly from navigator like I can do with epub files.

**Comment by OzzieIsaacs** [comment]
It's not perfect, but basically working
